### PR TITLE
Add vulkan pkgs

### PR DESCRIPTION
**Pull request by zhouleonlei**

### CODE REVIEW

**Review by wanchao-xu — APPROVED** [review]
Looks good to me

File: sysroot/build-rootfs.py [hunk 1]
```diff
@@ -105,6 +105,8 @@
     'libxkbcommon-devel',
     'tzsh',
     'tzsh-devel',
+    'vulkan-headers',
+    'vulkan-loader',
     'xdgmime',
     'xdgmime-devel',
     'wayland-extension-client-devel',
```
Comment (swift-kim) [inline sysroot/build-rootfs.py:112]: Actually `xdgmime*` should come after `wayland*`.

**Review by swift-kim — APPROVED** [review]
Are you working on Vulkan support? How is the progress and how much change (roughly) will it require to our embedder?

**Comment by zhouleonlei** [comment]
> Are you working on Vulkan support? How is the progress and how much change (roughly) will it require to our embedder?

Yes. It works on flutter-tizen 3.3.0 and about 600 lines codes. I will make a pull request for reviewing

File: sysroot/build-rootfs.py (hunk 1 above)
Comment (zhouleonlei) [reply sysroot/build-rootfs.py:112]: > Actually `xdgmime*` should come after `wayland*`.

For tizen 4.0 and 5.0, the vulkan pkgs names are 
Vulkan-LoaderAndValidationLayers and Vulkan-LoaderAndValidationLayers-devel

For tizen 5.5~7.0, the vulkan pkgs names are 
vulkan-headers vulkan-loader

Should I include "Vulkan-LoaderAndValidationLayers" for tizen 4.0 5.0?

Comment (swift-kim) [reply sysroot/build-rootfs.py:112]: No, we use only the Tizen 5.5 rootstrap for building the embedder. I was just talking about the alphabetical order of the packages which should not necessarily be fixed by this PR.

**Review by xiaowei-guan — APPROVED** [review]

**Review by AchoWang — COMMENTED** [review]
Looks good to me.

**Pull request merged**